### PR TITLE
Missed some parameters in recent sdk update.  Adding them

### PR DESCRIPTION
**Pull request by jayongg**

### CODE REVIEW

File: src/MicrosoftTeams.ts [hunk 1]
```diff
@@ -55,6 +55,7 @@ namespace microsoftTeams {
 
     export interface TabInstance {
         tabName: string;
+        tabId: string;
```
Comment (ydogandjiev) [inline src/MicrosoftTeams.ts]: Hmm, why are we exposing this? Not a good idea; this is an internal detail of Teams and this ID is subject to mutation which is not made obvious here

**Comment by jayongg** [comment]
Necessary for navigation to work.  We can document it as internal only. But my understanding is there is no way to navigate from the other data.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

File: src/MicrosoftTeams.ts (hunk 1 above)
Comment (tacri) [reply src/MicrosoftTeams.ts]: To navigate to a channel we need the channelId and the tabid. We can try to find the tabid when the method is invoked based on the tab properties passed and rely that there is only a tab with that name. 
Or we could also rename this to tabInstanceId

Comment (ydogandjiev) [reply src/MicrosoftTeams.ts]: As per our offline conversation we will indeed need this; however, I don't think we should expose it to the developers so it doesn't need to be on the interface.

**Comment by jayongg** [comment]
created a different PR - abandoning this one

**Pull request closed without merging**